source/js/components/RemainingPomos.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file Creates a custom element to display remaining planned pomos
+ * @author Meshach Adoe
+ * Date: 05/11/2022
+ */
+
+import * as backend from '../backend.js';
+import TaskList from '../classes/TaskList.js';
+import svgIcons from '../constants/themeIcons.js';
+
+/**
+ * Creates the HTML for the remaining pomo indicator
+ * @extends HTMLElement
+ */
+
+class RemainingPomos extends HTMLElement {
+  /**
+   * constructor for Remaining Pomo Indicator
+  */
+  constructor() {
+    super();
+    this.taskList = new TaskList();
```

Comment:
I think you can access this by doing backend.get('Tasklist') instead of creating a tasklist object